src/main/java/crazypants/enderio/config/Config.java

```diff
@@ -439,6 +446,13 @@ public String lc() {
   public static int soulBinderLevelOnePowerPerTickRF = 500;
   public static int soulBinderLevelTwoPowerPerTickRF = 1000;
   public static int soulBinderLevelThreePowerPerTickRF = 2000;
+  public static int soulBinderLevelFourPowerPerTickRF = 4000;
+  public static int soulBinderLevelFivePowerPerTickRF = 8000;
+  public static int soulBinderLevelSixPowerPerTickRF = 16000;
+  public static int soulBinderLevelSevenPowerPerTickRF = 32000;
```

Comment:
Why level 6 & 7 provide more power to the soulBinder than to the powered spawner (and in reverse order), is that intended?

Reply:
power spawner basic work time is random(200,800), so it use (work time)*(power per tick) rf. i set a capacitor >= tier 4 have a work time ran(25, 100) same to DE's spawner. so if spawner use more power in a tier 5&6&7 spawer, it just eat more power to do same work.
soulBinder's recipe use another logic. they use same power no matter capacitor tier. so higer power use is just a speed up

Reply:
instand of more speed , i choose higher tier(>=4)  capacitor to use less power  to spawn same amount mobs.

Reply:
that makes little sense tho: RFs are super easy to make, and saving RF costs at mid game or endgame seems weird. I would have thought that a capacitor upgrade would improve the perfs not save the consumption. That makes it less interesting endgame because why make expansive capacitor if you can do the lower one for the same speed?

Reply:
to be honest, i agree what you said. but how to improve it?

Reply:
just make it increase speed to keep the improvement logic, but make the RF input needed more consequent, so it doesn't become ridiculously cheap to power with last capacitors.